api/config.py

```diff
@@ -84,22 +85,22 @@
 ).get_default_database()
 log.debug(str(db))
 
+es = elasticsearch.Elasticsearch([__config['persistent']['elasticsearch_host']])
```

Comment:
Eh, I'm fine either way I guess.
Do we have a list somewhere of which config fields are pushed to the database but never read?